pom.xml

```diff
@@ -8,7 +8,7 @@
     </parent>
     <artifactId>entando-plugin-jpseo</artifactId>
     <groupId>org.entando.entando.plugins</groupId>
-    <version>7.1.1</version>
+    <version>7.1.3</version>
```

Comment:
I've just seen similar situations (moving from 7.1.0 to 7.1.2, skipping one version) with Breno yesterday. We agreed with Walter that for minor versions we increment the modules without skipping versions, so here you should put 7.1.2
This of course lead to mixed versions in de-app, see for example the de-app 7.1.2 release PR: https://github.com/entando/entando-de-app/pull/448/files